builtin/providers/nsone/resource_record.go

```diff
@@ -264,17 +269,25 @@ func recordToResourceData(d *schema.ResourceData, r *nsone.Record) error {
 		for regionName, region := range r.Regions {
 			newRegion := make(map[string]interface{})
 			newRegion["name"] = regionName
-			if len(region.Meta.GeoRegion) > 0 {
-				newRegion["georegion"] = region.Meta.GeoRegion[0]
+			// TODO: support as FeedPtr
```

Comment:
That seems nasty to have to do to every metadata field tho...

but im not sure of a cleaner way